packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -260,6 +290,18 @@ export default {
 
         isDisabled () {
             return this.$attrs.disabled === 'disabled';
+        },
+
+        isValidIconField () {
+            return VALID_ICON_INPUT_TYPES.includes(this.inputType);
+        },
+
+        hasLeftIcon () {
+            return this.$slots['icon-left'] && this.isValidIconField;
+        },
+
+        hasRightIcon () {
+            return this.$slots['icon-right'] && this.isValidIconField && !this.isDropdown;
```

Comment:
Same here – would be good to throw an error if someone tries to use a right icon with a dropdown – otherwise this handles the error but doesn't tell the consuming application why it's not rendering which can be a bit confusing.